components/common/Footer.tsx

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import { SNS_LIST } from '@/constants/default';
+import Link from 'next/link';
+import Image from 'next/image';
+import { DEVICE_SIZE } from '@/styles/DeviceSize';
+
+function Footer() {
+  return (
+    <Outer>
+      <Container>
+        <Resources>©codeit - 2023</Resources>
+        <TextBox>
+          <Text href="/privacy">Privacy Policy</Text>
+          <Text href="/faq">FAQ</Text>
+        </TextBox>
+        <SocialBox>
+          {SNS_LIST.map((sns) => (
```

Comment:
```suggestion
          {SNS_LIST.map(({ url, alt, src }) => (
```
이런식으로 destruction해서 가져오게되면 sns.url과 같이 접근하지 않아도 되겠죠? ㅎㅎ